wger/manager/models.py

```diff
@@ -725,6 +725,7 @@ class WorkoutLog(models.Model):
     '''
 
     date = Html5DateField(verbose_name=_('Date'))
+    session_id = models.ForeignKey('WorkoutSession', null=True, blank=True, to_field='id')
```

Comment:
@RrNn  you cannot name this `session_id` because in this case, it's an instance of a WorkoutSession.